### ISSUE TITLE
zombie lost when requesting raw data like images

### ISSUE THREAD

**Issue by passbe** [beefproject/beef#458]
<b>What steps will reproduce the problem?</b>
request an image like favicon.ico through the requester

<b>What is the expected output? What do you see instead?</b>
zombie detached from BeEF, the following error in JS console (Webkit/Gecko browsers):
Uncaught Error: INVALID_CHARACTER_ERR: DOM Exception 5
beef.encode.base64.encodehook.js:2784
beef.net.flushhook.js:2533
beef.net.sendhook.js:2526
(anonymous function)basic.html:19
(anonymous function)hook.js:2651
d.extend._Deferred.f.resolveWithhook.js:16
vhook.js:16
d.support.ajax.d.ajaxTransport.send.c

<b>Please use labels and text to provide additional information.</b>
beef.encode.base64.encode and decode are bugged. they don't play well with Unicode strings.<br><br><em>Google Code Issue: http://code.google.com/p/beef/issues/detail?id=354</em>
